samba/save_pshare.cgi

```diff
@@ -53,13 +53,13 @@ if ($name ne "global") {
 	foreach (&list_shares()) {
 	        $exists{$_}++;
 	        }
-	if (!$in{old_name} && $exists{$name}) {
+	if (!$in{'old_name'} && $exists{$name}) {
 	        &error(&text('savepshare_exist', $name));
 	        }
-	elsif ($in{old_name} ne $name && $exists{$name}) {
+	elsif ($in{'old_name'} ne $name && $exists{$name}) {
 	        &error(&text('savepshare_exist', $name));
 	        }
-	elsif ($name !~ /^[A-Za-z0-9_\$\-\. ]+$/) {
+	elsif ($name !~ /^[\p{L}\p{N}_\$\-\.\s]+$/) {
```

Comment:
This particular change is meant to be the part of https://github.com/webmin/webmin/pull/2025/commits/59b944ebff964c73f8b57443f6799cb0065c7dcc commit of this PR!